projects/ngx-translate/core/src/lib/translate.service.ts

```diff
@@ -365,30 +363,15 @@ export class TranslateService {
     }
     // check if we are loading a new translation to use
     if (this.pending) {
-      return Observable.create((observer: Observer<string>) => {
-        let onComplete = (res: string) => {
-          observer.next(res);
-          observer.complete();
-        };
-        let onError = (err: any) => {
-          observer.error(err);
-        };
-        this.loadingTranslations.subscribe((res: any) => {
```

Comment:
Making subscriptions inside `new Observable.()` is almost never neccassy. It's also kind of a bad practise because it makes the outer Observable unsubscribable.
It seemed to me that this could be simplified significantly.